packages/grafana-ui/src/components/ServiceMap/GraphViewCy.tsx

```diff
@@ -0,0 +1,214 @@
+import { fromPairs } from 'lodash';
+import React, { useEffect, useState } from 'react';
+import { SimulationNodeDatum, SimulationLinkDatum } from 'd3-force';
+import useMeasure from 'react-use/lib/useMeasure';
+import { response } from './x-ray-response';
+import cytoscape, { ElementDefinition } from 'cytoscape';
```

Comment:
how big a framework is this? We already use another network graph library (visjs-network), is there a way to use the same? (or update that to use this) 

Used here
https://github.com/grafana/grafana/blob/master/public/app/features/variables/inspect/NetworkGraph.tsx

Reply:
Right now there is a bit more of these as I was testing different libraries and layouts. In the end this will be removed and we will probably go just with the d3-force layout.